docs/installation.md

```diff
@@ -42,15 +42,16 @@ az aks install-cli
 Install the Workspace controller.
 
 ```bash
-helm install workspace ./charts/kaito/workspace --namespace kaito-workspace --create-namespace
+export KAITO_WORKSPACE_VERSION=0.4.2
```

Comment:
why changed this? we can assume user are working on the target git branch. then just apply manifests from repo.

Reply:
There is a comment: https://github.com/kaito-project/kaito/pull/823#pullrequestreview-2553575559, and I update the installation in this pull request.